### DISCUSSION TITLE
Editor Improvement

### DISCUSSION THREAD

**Discussion by eta-git** [ppy/osu#31199] [Ideas] [answered]
The current osu lazer editor doesn't support placing notes in 1/5, 1/6, 1/7, and so on. Will that ever be implemented on lazer?

**Comment by cihe13375** [comment, marked as answer]
1/6 (and triplets in general) can be accessed by switching to the "triplets" mode (click the arrows near the "common" text)

arbitrary divisors can be accessed by clicking on the "1/4" (or whatever number that is currently active) text (I only realized it after searching it and found the pr https://github.com/ppy/osu/pull/17013)
